ro_help/locale/ru/LC_MESSAGES/django.po

```diff
@@ -0,0 +1,2036 @@
+msgid "Register"
+msgstr "Регистрация"
+
+#: hub/templates/incubated-by.html:7
+msgid "Project developed in the program"
+msgstr "Проэкт разработан в программе"
+
+#: hub/templates/ngo/base.html:11
+msgid "Search..."
+msgstr "Поиск..."
+
+#: hub/templates/ngo/base.html:27
+msgid "Welcome to RoHelp"
+msgstr "Добро пожаловть в Ajut.md"
+
+#: hub/templates/ngo/base.html:30
+msgid ""
+"RoHelp is a fully-featured digital platform which lends itself to be used by "
+"all organizations which are actively involved in halting the spread of "
+"Covid-19. The main purpose of the paltform is to help organizations collect "
+"the resources they need. As a user, the platform will help you find the most "
+"pressing needs that organizations face across the country. As a result, it "
+"makes it easy for you to chose the cause you donate to, or the efforts you "
+"contribute to."
+msgstr ""
+"Ajut.md - это полнофункциональная цифровая платформа, которая может быть использована"
+"всеми организациями, которые активно участвуют в предотвращении распространения"
+"Covid-19. Основная цель палтформы - помочь организациям собрать"
+"ресурсы, которые им нужны. Платформа поможет вам найти больше всего"
+"насущные потребности, с которыми сталкиваются организации по всей стране. В результате, это"
+"облегчает вам выбор дела, которому вы жертвуете, или усилий, которые вы делаете"
+"способствовать."
+
+#: hub/templates/ngo/base.html:47
+msgid "I want to help with money"
+msgstr "Я хочу помочь деньгами"
+
+#: hub/templates/ngo/base.html:59
+msgid "I want to help with resources"
+msgstr "Я хочу помочь ресурсами"
+
+#: hub/templates/ngo/base.html:71
+msgid "I want to volunteer"
+msgstr "Я хочу стать волонтером"
+
+#: hub/templates/ngo/base.html:97
+msgid "Red Cross"
+msgstr "Крассный крест"
+
+#: hub/templates/ngo/base.html:100
+msgid ""
+"\n"
+"                            Societatea Națională de Cruce Roșie Română "
+"reprezintă punctul principal de colectare la nivel național, atât pentru "
+"donații financiare care vor fi utilizate în vederea achiziționarii de "
+"echipamente și materiale sanitare, respectiv produse și bunuri materiale "
+"care vor fi distribuite - în funcție de nevoi și/sau solicitări - pe "
+"teritoriul național.\n"
+"                            "
+msgstr ""
+"Молдавская национальное общество Красного Креста"
+"представляет собой главный пункт сбора на национальном уровне, как для"
+"финансовые пожертвования на покупку"
+"сантехническое оборудование и материалы, соответственно изделия и материальные товары"
+"которые будут распространяться - в зависимости от потребностей и/или запросов - на"
+"национальная территории.\n"
+""
+
+#: hub/templates/ngo/base.html:109 hub/templates/ngo/detail.html:100
+#: hub/templates/ngo/donate.html:26 hub/templates/ngo/list.html:245
+msgid "Donate"
+msgstr "Пожертвование"
+
+#: hub/templates/ngo/detail.html:36
+msgid "donations"
+msgstr "Пожертвования"
+
+#: hub/templates/ngo/detail.html:41
+msgid "RON donated"
```

Comment:
Please change here to _MDL donated_
Because this was changed in another PR and merged.